pkg/image/api/helper.go

```diff
@@ -165,6 +165,21 @@ func (r DockerImageReference) String() string {
 	return fmt.Sprintf("%s%s%s", registry, r.Namespace, r.NameString())
 }
 
+// StringNoTag converts a DockerImageReference to a Docker pull spec with no tag.
+func (r DockerImageReference) StringNoTag() string {
+	registry := r.Registry
+	if len(registry) > 0 {
+		registry += "/"
+	}
+
+	if len(r.Namespace) == 0 {
```

Comment:
I noticed :) We should change both. We should at least be internally consistent... if we use a local var for the registry then we should use one for the namespace. Or don't for both.